tests/reactable_test.jsx

```diff
@@ -1608,6 +1608,51 @@ describe('Reactable', function() {
             });
         });
 
+        describe('yen sort', function(){
+            before(function() {
+                ReactDOM.render(
+                    <Reactable.Table className="table" id="table" data={[
+                        { Price: '1.25'},
+                        { Price: '¥1.01'},
+                        { Price: '1'},
+                        { Price: '¥10,000'},
+                        { Price: '¥10,500'},
+                        { Price: '¥10'},
+                        { Price: 'a'},
+                        { Price: 'z'},
+                        { Price: '¥2'},
+                        { Price: '¥.5'},
+                        { Price: '¥0.60'},
+                        { Price: '.1'},
+                    ]}
+                    columns={[{ key: 'Price', sortable: Reactable.Sort.Currency }]} />,
+                    ReactableTestUtils.testNode()
+                );
+            });
+
+            after(function() {
+                ReactableTestUtils.resetTestEnvironment();
+            });
+
+            it('sorts columns numerically. parsing out currency symbols', function(){
+                var sortHeader = $('#table thead tr.reactable-column-header th')[0];
+                ReactTestUtils.Simulate.click(sortHeader);
+
+                ReactableTestUtils.expectRowText(0, ['.1']);
+                ReactableTestUtils.expectRowText(1, ['¥.5']);
+                ReactableTestUtils.expectRowText(2, ['¥0.60']);
+                ReactableTestUtils.expectRowText(3, ['1']);
+                ReactableTestUtils.expectRowText(4, ['¥1.01']);
+                ReactableTestUtils.expectRowText(5, ['1.25']);
+                ReactableTestUtils.expectRowText(6, ['¥2']);
+                ReactableTestUtils.expectRowText(7, ['¥10']);
+                ReactableTestUtils.expectRowText(8, ['¥10,000']);
+                ReactableTestUtils.expectRowText(9, ['¥10,500']);
+                ReactableTestUtils.expectRowText(10, ['a']);
+                ReactableTestUtils.expectRowText(11, ['z']);
+            });
+        });
```

Comment:
this was the quickest way for me to get a breaking test. currently thinking of ways to optimize this, but for now it works. open to input, but otherwise i'll replace this commit once i think of something better.